cvat/apps/dataset_manager/tests/assets/annotations.json

```diff
@@ -0,0 +1,734 @@
+{
```

Comment:
I'm both hands for using data-driven approach, but think how we can optimize the size of testing code and setup. It is very hard to reason about the tests and their goals in the current setup. Please also note that different formats can require or can utilize specific task settings like existence of specific attributes and specific values. Several formats can accept different input file layouts (CVAT - an xml instead of zip, COCO - a json instead of zip, PASCAL - a zip with a bunch of xmls, [more info here](https://github.com/openvinotoolkit/cvat/blob/develop/cvat/apps/dataset_manager/formats/README.md)).

Reply:
Tests use only zip format files. If annotations formats support attributes then they random generate, but if they do not support then used default values atributes.

Reply:
I didn't gather why tests do not test the provided functionality in formats?